.github/workflows/build_and_test.yaml

```diff
@@ -134,6 +134,7 @@ jobs:
             sudo dpkg -l | grep -i mysql
             sudo apt-get clean
             sudo apt-get install -y mysql-client
+            mysql --host 127.0.0.1 --port 3306 -uroot -ppassword -e "SET GLOBAL max_connections = 2000"
```

Comment:
are you able to tell whether tests are passing due to this increase or because of better hygiene of closing connections/statements?